tests/tensor_search/test_get_stats.py

```diff
@@ -14,16 +14,10 @@ class TestGetStats(MarqoTestCase):
     def setUp(self) -> None:
         self.generic_header = {"Content-type": "application/json"}
         self.index_name_1 = "my-test-index-1"
-        try:
-            tensor_search.delete_index(config=self.config, index_name=self.index_name_1)
-        except IndexNotFoundError as s:
-            pass
+        self.clear_index_by_name(self.index_name_1)
 
     def tearDown(self) -> None:
-        try:
-            tensor_search.delete_index(config=self.config, index_name=self.index_name_1)
-        except IndexNotFoundError as s:
-            pass
+        self.clear_index_by_name(self.index_name_1)
```

Comment:
`tensor_search.get_stats` has been replaced by `monitoring` module. These tests should be failing to run? I think they're redundant now and go (or move to monitoring tests, if there's anything here that isn't covered there)

Reply:
These tests are skipped for now, I will check for redundancies and remove this file